share/pvss/pvss.go

```diff
@@ -152,16 +184,22 @@ func DecShareBatch(suite Suite, H kyber.Point, X []kyber.Point, sH []kyber.Point
 // log_{G}(X) == log_{sG}(sX). Note that X = xG and sX = s(xG) = x(sG).
 func VerifyDecShare(suite Suite, G kyber.Point, X kyber.Point, encShare *PubVerShare, decShare *PubVerShare) error {
 	if err := decShare.P.Verify(suite, G, decShare.S.V, X, encShare.S.V); err != nil {
-		return errorDecVerification
+		return errDecVerification
```

Comment:
Can we change the signature of this function to have a bool? 

Using errors to Verify cryptography is a bad idea:

https://www.youtube.com/watch?v=4opI0w0Xs5c

Reply:
the other option would be to export that error and them wrap it with `fmt.Errorf("invalid: %w", ErrDecVerification)` so that the error cannot be "bypassed" but it's still possible to match on it.